test/framework/log/collector.go

```diff
@@ -96,6 +96,8 @@ func (c *MachineLogCollector) CollectMachineLog(ctx context.Context, ctrlClient
 			"sudo", "cat", "/var/log/cloud-init.log"),
 		captureLogs("cloud-init-output.log",
 			"sudo", "cat", "/var/log/cloud-init-output.log"),
+		captureLogs("kubeadm-service.log",
```

Comment:
I assume this doesn't fail the test because we only bubble up the error to a certain point, but we don't fail the test because of that
(also the reason why the cloud-init files above don't lead to a failure of the ignition test)

Reply:
Exact, it does not fail on errors, just logs it.